meteor/lib/Settings.ts

```diff
@@ -87,6 +93,7 @@ const DEFAULT_SETTINGS = Object.freeze<ISettings>({
 	useCountdownToFreezeFrame: true,
 	confirmKeyCode: 'Enter',
 	customizationClassName: 'tv2',
+	interpolateStringsInLayouts: false,
```

Comment:
Shouldn't this be true? These are our defaults for settings for Sofie. Doesn't make much sense to add a setting for then not to use it.

Reply:
> These are our defaults for settings for Sofie.

No, these are not

Reply:
Does this need a setting? If so I would prefer this setting to be something on the showstyle the panels belong to (or each panel individually?) rather than hidden away here. Having it here massively reduces discoverability of the feature.

I expect that we can make use of this at NRK

Reply:
> If you want them to be, we need consensus on getting rid of core-settings.json

I thought we had consensus on that allready?

Reply:
@Julusian okay, well if you think this is something that could be enabled by default, I'll happily get rid of the setting. I thought we would only enable it in the installations where it's needed, but It's not like `replace` on a few relatively short strings will affect performance, right?
@LindvedKrvang I think [this](https://github.com/tv2/tv-automation-server-core/pull/224#discussion_r1151483076) is where we're currently at

Reply:
I think we should have a talk about it with the team then :)